psi4/src/psi4/lib3index/df_helper.cc

```diff
@@ -2239,16 +2932,21 @@ void DF_Helper::compute_K(std::vector<SharedMatrix> Cleft,
         size_t cright = Cright[i]->colspi()[0];
         double* Kp = K[i]->pointer()[0];
 
+        // cache alignment (is this over-zealous?)
```

Comment:
Admittedly for larger systems the benefit is nominal.  The C_buffer alignment isnice as it completely partitions resources for threads when the ends of O(np) matrices share a cache line.  However this is for false sharing and not vectorization and these overlaps will become a tiny fraction for large n.